services/wishlistService.js

```diff
@@ -0,0 +1,24 @@
+const wishlistDao = require('../models/wishlistDao');
+
+const getWishlist = async(userId) => {
+  return await wishlistDao.getWishlist(userId);
+};
+
+const deleteWishlist = async(userId, productId) => {
+  return await wishlistDao.deleteWishlist(userId, productId);
+};
+
+const addWishlist = async(userId, productId) => {
+  const checkWishlist = await wishlistDao.checkWishlist(userId, productId)
+  if(!checkWishlist.length){
+     await wishlistDao.addWishlist(userId, productId);
+  } else{
+     await wishlistDao.deleteWishlist(userId, productId);
```

Comment:
`wishlistDao.deleteWishList()`가 반환하는 값이 있으니, service에서도 해당 값을 반환하여 controller에서도 해당 값을 받을 수 있게 수정해주세요.
```suggestion
     return await wishlistDao.deleteWishlist(userId, productId);
```